2024-06-14 09:30:49.746826: I tensorflow/core/platform/cpu_feature_guard.cc:151] This TensorFlow binary is optimized with oneAPI Deep Neural Network Library (oneDNN) to use the following CPU instructions in performance-critical operations:  SSE4.1 SSE4.2 AVX AVX2 FMA
To enable them in other operations, rebuild TensorFlow with the appropriate compiler flags.
2024-06-14 09:30:51.782759: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1525] Created device /job:localhost/replica:0/task:0/device:GPU:0 with 10438 MB memory:  -> device: 0, name: NVIDIA GeForce GTX 1080 Ti, pci bus id: 0000:83:00.0, compute capability: 6.1
Traceback (most recent call last):
  File "/data/leslie/sarthak/chrombpnet/chrombpnet/CHROMBPNET.py", line 181, in <module>
    main()
  File "/data/leslie/sarthak/chrombpnet/chrombpnet/CHROMBPNET.py", line 25, in main
    pipelines.chrombpnet_train_pipeline(args)
  File "/data/leslie/sarthak/chrombpnet/chrombpnet/pipelines.py", line 85, in chrombpnet_train_pipeline
    train.main(args_copy)
  File "/data/leslie/sarthak/chrombpnet/chrombpnet/training/train.py", line 92, in main
    train_generator = initializers.initialize_generators(args, "train", parameters, return_coords=False, save_data=save_data)
  File "/data/leslie/sarthak/chrombpnet/chrombpnet/training/data_generators/initializers.py", line 80, in initialize_generators
    generator=batchgen_generator.ChromBPNetBatchGenerator(
  File "/data/leslie/sarthak/chrombpnet/chrombpnet/training/data_generators/batchgen_generator.py", line 36, in __init__
    peak_seqs, peak_cts, peak_coords, nonpeak_seqs, nonpeak_cts, nonpeak_coords, = data_utils.load_data(peak_regions, nonpeak_regions, genome_fasta, cts_bw_file, inputlen, outputlen, max_jitter, save_data)
  File "/data/leslie/sarthak/chrombpnet/chrombpnet/training/utils/data_utils.py", line 95, in load_data
    np.save(save_data+"peaks_seqs.npy", train_peaks_seqs)
  File "<__array_function__ internals>", line 180, in save
  File "/data/leslie/sarthak/environments/chrombpnet/lib/python3.10/site-packages/numpy/lib/npyio.py", line 498, in save
    file_ctx = open(file, "wb")
FileNotFoundError: [Errno 2] No such file or directory: '/data/leslie/sarthak/data/chrombpnet_test/chrombpnet_data_1000/peaks_seqs.npy'
